src/EFCore.PG/Infrastructure/Internal/NpgsqlOptionsExtension.cs

```diff
@@ -91,7 +82,6 @@ public class NpgsqlOptionsExtension : RelationalOptionsExtension
         public NpgsqlOptionsExtension()
         {
             _rangeMappings = new List<RangeMappingInfo>();
-            _plugins = new List<NpgsqlEntityFrameworkPlugin>();
```

Comment:
With this gone, you could go to an expression-bodied constructor here.

Reply:
Done